tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
```

Comment:
Calling a util function for logging is ok. It helps reduce the clutter in actual code (where we create event class etc).

@anmolsachan please ensure this util is generic enough to handle all our logging scenarios. Provide comments related to that here